### PR TITLE
Feat/responsive ads

### PR DESCRIPTION
**Pull request by killanaca**
Add support for responsive ads (different ads size for different breakpoint), no different bidding.
Note: branch off  from`feat/prebid-rebase`

### CODE REVIEW

File: packages/ad/ad.stories.js [hunk 1]
```diff
@@ -96,7 +96,7 @@ storiesOf("Advertisement", module)
           Orci varius natoque penatibus et magnis dis parturient montes,
           nascetur ridiculus mus.
         </Text>
-        <Ad pos="header" section="news" contextUrl={articleUrl} />
+        <Ad pos="intervention" section="news" contextUrl={articleUrl} />
```
Comment (BernieSumption) [inline packages/ad/ad.stories.js]: "intervention" - not a valid pos? It's not in slot-options.json anyway. I think this is supposed to be "inline-ad". Other stories have it too, e.g. "render one ad - intervention".

Comment (nehasri89) [reply packages/ad/ad.stories.js]: @BernieSumption  it is a valid pos, I have checked in render. We should update the fixture for it, it has the size mapping for it and acts as a default ad size category

Comment (nehasri89) [reply packages/ad/ad.stories.js]: ok, I checked render again. `intervention` is only ever use as a default size mapping, and you are right it is not a valid pos @BernieSumption  👍

**Comment by killanaca** [comment]
superseded by https://github.com/newsuk/times-components/pull/733

**Pull request closed without merging**